panopticum/templates/registration/login.html

```diff
@@ -20,7 +20,7 @@
               {% csrf_token %}
               <h1>Login Form</h1>
               <div>
-                <input type="text" name="username" class="form-control" placeholder="Username" required="" />
+                <input type="text" name="username" class="form-control" placeholder="john.doe" required="" />
```

Comment:
Are we sure we need 'john.doe'?

Reply:
Some users ask about username format. Do we need domain or email domain suffix